packages/suite/src/storage/migrations/versions/25.8.0.ts

```diff
@@ -1,11 +1,35 @@
 import { createMigration } from '@suite/idb-migration-utils';
 
+import { LANGUAGES } from 'src/config/suite';
+import type { Locale } from 'src/config/suite/languages';
 import { SuiteDBSchema } from 'src/storage/definitions';
 
 import { updateAll } from '../utils';
 
-export default createMigration<SuiteDBSchema>('25.8.0', async (_db, transaction) => {
-    await updateAll(transaction, 'devices', device => {
+export default createMigration<SuiteDBSchema>('25.8.0', async (db, tx) => {
+    if (db.objectStoreNames.contains('suiteSettings')) {
+        const store = tx.objectStore('suiteSettings');
+        const settings = await store.get('suite');
+
+        if (settings && settings.settings) {
+            if (
+                typeof settings.settings.language === 'string' &&
+                settings.settings.language.length === 2
+            ) {
+                const languageKeys = Object.keys(LANGUAGES) as Locale[];
```

Comment:
`import { typedObjectKeys } from '@trezor/utils';`
```suggestion
                const languageKeys = typedObjectKeys(LANGUAGES);
```

Reply:
nice! totally forgot about our typedObjectKeys

Reply:
fixed